packages/web-components/fast-foundation/src/data-grid/data-grid-cell.template.ts

```diff
@@ -0,0 +1,16 @@
+import { html } from "@microsoft/fast-element";
+import { DataGridCell } from "./data-grid-cell";
+
+/**
+ * The template for the {@link @microsoft/fast-foundation#DataGridCell} component.
+ * @public
+ */
+export const DataGridCellTemplate = html<DataGridCell>`
+    <template
+        tabindex="-1"
+        role="${x => (x.cellType === "columnheader" ? "columnheader" : "gridcell")}"
+        class="${x => (x.cellType === "columnheader" ? "column-header" : "")}"
+    >
+        <slot part="cellSlot"></slot>
```

Comment:
I believe we landed spinal case for HTML attributes and values.

Reply:
So "cellRegion"?

Reply:
I have been seeing spinal case for part names like `expanded-icon`, `collapsed-icon`

Reply:
@scomea I think "cell-region" might work here.

Reply:
Actually, if this is the component's default slot does it need a part name?

Reply:
@chrisdholt @nicholasrice can someone answer this? I've no idea.

Reply:
No name is needed if it's intended to be the default slot. Also, we'd want to always use the default slot before adding named slots.

Reply:
Oops. Sorry, I misread. I don't know if a `part` name directly on a slot will have any effect. I'm not sure if you can style that slot directly. If it is possible, and it's the default slot then perhaps we should call the part itself "default" as well?

Reply:
The part should go on the same level as the class. The intent here is that we style with the class, someone can reach in and change that with the part. There are plenty of methods to style that slotted content from a user perspective. The part itself should be on the DOM inside the component rather than the slot itself.

Reply:
Ok, so since the slot the single child of the template root no need for a partname